### ISSUE TITLE
SSL - Connect failed -13

### ISSUE THREAD

**Issue by TDERL** [Wiznet/RP2040-HAT-C#11]
Hi,
Hoping someone can help.
Trying to run the TCP Client over SSL example. Run through the steps as described. I get my openSSL to the point where it displays:

Using default temp DH parameters
ACCEPT

Then I reset my RP2040 Pico board and get:

====================================================================================================<CR><LF>
 W5100S network configuration : static<CR><LF>
<CR><LF>
 MAC         : nn:nn:nn:nn:nn:nn<CR><LF>
 IP          : 192.168.n.nnn<CR><LF>
 Subnet Mask : 255.255.255.0<CR><LF>
 Gateway     : 192.168.n.n<CR><LF>
 DNS         : 8.8.8.8<CR><LF>
====================================================================================================<CR><LF>
<CR><LF>
 Socket descriptor 0<CR><LF>
 Supported ciphersuite lists<CR><LF>
 TLS-DHE-RSA-WITH-AES-256-CBC-SHA256       <CR><LF>
 TLS-DHE-RSA-WITH-AES-128-GCM-SHA256<CR><LF>
 TLS-DHE-RSA-WITH-AES-128-CBC-SHA256       <CR><LF>
 TLS-RSA-WITH-AES-256-CBC-SHA256<CR><LF>
 TLS-RSA-WITH-AES-128-GCM-SHA256           <CR><LF>
 TLS-RSA-WITH-AES-128-CBC-SHA256<CR><LF>
 Connect failed -13<CR><LF>

I can see that error -13 is a timeout, but I can work out why or how to debug it. Any pointers would be very gratefully accepted!

Many thanks

**Comment by GiungKim** [comment]
Hello @TDERL,

'Connect failed -13' is a return value that occurs as a timeout because it cannot connect to the SSL/TLS server.

Did you set the SSL/TLS server IP correctly?

'g_ssl_target_ip' is the setting for the SSL/TLS server IP.

**Comment by TDERL** [comment]
Hi GiungKim,

As far as I can tell, yes. It's not easy to spot if I am doing one of the steps slightly wrong as the instructions are not complete and I am not very fluent with CLI's. I tried 'g_ssl_target_ip' but it is not recognised as a command, so am not sure how to use it or what I am doing wrong.

For now, I have moved onto other things as can't afford to waste time on things that don't work (which has been my general experience with most things RP2040 related). Perhaps I'll give it another try if/when I get a spare moment.
